cluster-autoscaler/core/utils.go

```diff
@@ -48,6 +49,20 @@ const (
 	ReschedulerTaintKey = "CriticalAddonsOnly"
 )
 
+var (
```

Comment:
We should consider adding a feature flag to disable/enable dropping standard taints.
I personally do not see a scenario where we would like to NOT drop them. But maybe I am missing something.
@MaciekPytel WDYT?

Reply:
I don't see any scenario for keeping those taints either. You should never use those specific taint keys for anything else, whether you use taint by condition or not. I don't think we need a flag to disable that logic (also - we don't have flag that would prevent CA from ignoring rescheduler taint, so there is precedent).

Reply:
Yeah I thought about this too and arrived at the same rationale for not providing a flag.